### PR TITLE
 Refactor repositories role to use variables to configure repos

### PR DESCRIPTION
**Pull request by jpward1981**

### CODE REVIEW

File: molecule/default/playbook.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+- name: Converge
+  hosts: all
+  vars:
+    - repos:
```
Comment (ragingpastry) [inline molecule/default/playbook.yml]: I believe this should be `repositories` instead of `repos`

File: molecule/default/playbook.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        redhat:
+          - { url: "https://centos7.iuscommunity.org" }
```
Comment (ragingpastry) [inline molecule/default/playbook.yml]: This doesn't appear to work on centos. Running `molecule converge` fails and logging into the container manually will result in the same error "No package https://centos7.iuscommunity.org available."

File: molecule/default/tests/test_default.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
+
+
+def test_hosts_file(host):
+    f = host.file('/etc/hosts')
+
+    assert f.exists
+    assert f.user == 'root'
+    assert f.group == 'root'
```
Comment (ragingpastry) [inline molecule/default/tests/test_default.py:14]: Can we add some simple testinfra tests? Ensuring that we can download/install a package provided by one of the configured repos should do the trick

File: meta/main.yml [hunk 4]
```diff
@@ -6,18 +7,9 @@ galaxy_info:
   min_ansible_version: 2.0
```
Comment (ragingpastry) [inline meta/main.yml]: We should probably bump this up at some point or test against all ansible versions between this and latest. This should be included in this PR, more just making note for later

**Review by ragingpastry — CHANGES_REQUESTED** [review]

File: molecule/default/playbook.yml (hunk 1 above)
Comment (jpward1981) [reply molecule/default/playbook.yml]: You are correct, I have updated this in the latest PR

**Review by ragingpastry — CHANGES_REQUESTED** [review]
Squash up some of these commits and we should be good to go!

**Review by ragingpastry — APPROVED** [review]

**Pull request merged**